karma.py

```diff
@@ -47,11 +47,22 @@ def karma(self, event):
     @cmd
     def top_karma(self, event):
         """Get 5 people with most karma points."""
+        args = event.args
         output = ""
         inverted = [(value, key) for (key, value) in self.karmas.items()]
         karmees = sorted(inverted, reverse=True)
-        # Takes top 5 or less if len(karmees) < 5
-        karmees = karmees[:5]
+
+        if len(args) == 0:
+          # Takes top 5 or less if len(karmees) < 5
+          karmees = karmees[:5]
+
+        if len(args) == 1:
+          # Takes top 'n' or less if len(karmees) < n
+          try:
+            n = int(args[0])
+          except ValueError:
+            return "Argument must be a number!"
+          karmees = karmees[:n]
```

Comment:
@JakubNvk this won't work if you do not set n to some value prior to using it here.